CONTRIBUTING.md

```diff
@@ -245,13 +251,27 @@ If in doubt, you can always ask for guidance in the Pull Request or on
 Feel free to post a comment in the Pull Request to ping reviewers if you are
 awaiting an answer on something.
 
+Notes: if the reviewer mentions *nits* in their comments, that stands for
+"requests for small changes that are not essential". Usually these nits are
+related to coding styles. It's always a good idea to pay attention to the
+detail!
 
 ### Step 8: Landing
 
-Once your Pull Request has been reviewed and approved by at least one Node.js
-Collaborators (often by saying LGTM, or Looks Good To Me), and as long as
-there is consensus (no objections from a Collaborator), a
-Collaborator can merge the Pull Request . GitHub often shows the Pull Request as
+A Pull Request needs to be reviewed and approved by at least one Node.js
+Collaborators (often by saying LGTM, or Looks Good To Me) to get landed.
```

Comment:
Does adding a link to the video tutorial(https://www.youtube.com/watch?v=HW0RPaJqm4g) and the documentation(https://help.github.com/articles/reviewing-changes-in-pull-requests/) suffice?